src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -35,12 +39,167 @@
+            // @attribute <mythicmobs.skills>
+            // @returns ListTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of valid MythicSkill IDs.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "skills", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (String item : getSkillNames()) {
+                    list.addObject(new ElementTag(item, true));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.mob_ids>
+            // @returns ListTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of valid MythicMob IDs.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "mob_ids", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (String item : getMobManager().getMobNames()) {
+                    list.addObject(new ElementTag(item, true));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.active_mobs>
+            // @returns ListTag(MythicMobsMobTag)
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of all active MythicMobs on the server.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "active_mobs", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (ActiveMob entity : getMobManager().getActiveMobs()) {
+                    list.addObject(new MythicMobsMobTag(entity));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.spawners>
+            // @returns ListTag(MythicSpawnerTag)
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of all MythicSpawners.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "spawners", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (MythicSpawner spawner : MythicMobsBridge.getSpawnerManager().getSpawners()) {
+                    list.addObject(new MythicSpawnerTag(spawner));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.damage_modifiers[<mob_id>]>
+            // @returns MapTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a map of the damage modifiers for a MythicMob from a Mob ID.
+            // -->
+            tagProcessor.registerTag(MapTag.class, ElementTag.class, "damage_modifiers", (attribute, object, id) -> {
+                Optional<MythicMob> optionalMythicMob = MythicBukkit.inst().getMobManager().getMythicMob(id.asString());
```

Comment:
Can use `getMobManager()` instead of getting it directly, or use the existing `getMythicMob` method